### DISCUSSION TITLE
3D-Messen

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#20] [Ideas]
Ist eine 3D-Szene fertig erstellt, wäre es super, wenn es eine Art von Downloadmöglichkeit (Bild) geben würde.

**Comment by gis-eni** [comment]
Das 3D Tool ist eigentlich ein "Meßtool". Die Darstellung und das Messen aus ausschließlich Clientseitig. Ein Drucken wird so schwierig. Wenn es nur um ein Bild geht, kann man als Anwender natürlich leicht einen Screenshot machen (SnipIt...)

Eine Download/Druck Möglichkeit wird daher derzeit nicht angedacht.
